### ISSUE TITLE
[Feature request] More extensive config

### ISSUE THREAD

**Issue by CvoidWyvern** [Melonslise/Subterranean-Wilderness#19]
Been trying to use this mod in conjunction with a bunch of other underground generation mods with heavy config tweaking and have been running into the problem of not being able to change much about how this mod generates. Also noticed that a lot of the issues here include requests for more config options.

Something along the lines of:
Disabling or lowering the chance of specific decorations (would be nice to be able to do this per cave type)
Adjusting the chance of each cave type
Adjusting the chance of a cave being decorated at all (could just be done with a cave type with no decorations with a default chance of 0, or multiple cave types like this for different heights)

**Comment by PenumbraStar** [comment]
May I also make the suggestion of allowing the disabling of the water pools for shaders and alternate world generators like Terraforged. The reason is because with shaders that make water transparent this is not affected and that Mods that greatly alter worldgen like Terraforged result in large amounts of water pools that look strange, other than that this mod is amazing for improving cave gen, I just wish to use it with Terraforged due to its awesome looking worldgen and this one thing looks extremely out of place.

**Comment by arBorghi** [comment]
Also would be awesome an option to avoid decorations with non vanilla blocks to avoid the amount of ram used.

**Comment by PadLogic** [comment]
it is possible to turn off foxfire mushroom generation?

**Comment by PaintNinja** [comment]
@PadLogic Yes, will be in the next update (https://github.com/Melonslise/Subterranean-Wilderness/commit/a55b3664af007c1a256184f5edf1429c01d6e254)

**Comment by Reshy** [comment]
The ability to scale down Speleothem density, as some biomes (like Jungle) are flooded with them to the point where you can't move through most caves.  Might also be a bug.
Ability to scale down the amount of dirt, mossy stone, molten stone, etc. replacement.  Some areas are almost completely covered in dirt, so you can't actually tell if there's ores there or not.
Ability to craft the stackable carpets of stone, dirt, etc, as they can make for nice decoration blocks.

**Comment by egshels** [comment]
The ability to disable the custom ore generation, or be compatible with something like Emendatus Enigmatica, would also be huge

**Comment by Rastang94** [comment]
For me is really important to be able to modify the height at which decorations will be applied, or having different type of decorations for lowers levels. This because on lowers levels at the moment i have huge lakes of lava (Yung's better caves mod) surrounded by ice (which melt and flood everything), or full of vines on fire... was a beautiful view the first time but i don't really like it.
I would like to have superficial caves that mimic the surface biome, and under a certain point (something like y30-40) normal caves, or something suitable for more hot and dangerous caves.
